src/status_im/ui/screens/profile/components/views.cljs

```diff
@@ -51,46 +48,55 @@
                 :ellipsize-mode  :tail}
     name]])
 
-(defn- profile-header-display [{:keys [name public-key] :as contact}
-                               allow-icon-change? include-remove-action?]
-  [react/view (merge styles/profile-header-display {:padding-horizontal 16})
-   (if allow-icon-change?
-     [react/view {:align-items     :center
-                  :align-self      :stretch
-                  :justify-content :center}
-      [react/touchable-highlight
-       {:accessibility-label :edit-profile-photo-button
-        :on-press
-        #(re-frame/dispatch
-          [:bottom-sheet/show-sheet
-           {:content        (sheets/profile-icon-actions include-remove-action?)
-            :content-height (if include-remove-action? 192 128)}])}
-       [react/view
-        [react/view {:background-color colors/white
-                     :border-radius    15
-                     :width            30
-                     :height           30
-                     :justify-content  :center
-                     :align-items      :center
-                     :position         :absolute
-                     :z-index          1
-                     :top              -5
-                     :right            -5}
-         [react/view {:background-color colors/blue
-                      :border-radius    12
-                      :width            24
-                      :height           24
+(defn chat-key-popover [public-key ens-name]
+  (re-frame/dispatch [:show-popover
+                      {:view :share-chat-key
+                       :address public-key
+                       :ens-name ens-name}]))
+
+(defn- profile-header-display
+  [{:keys [name public-key ens-name] :as contact}
+   allow-icon-change? include-remove-action?]
+  [react/touchable-opacity {:on-press #(chat-key-popover public-key ens-name)}
```

Comment:
I would have used [:<> but it looks like we are still using a deadass version of reagent